net-p2p/qbittorrent/qbittorrent-3.3.4.recipe

```diff
@@ -52,7 +57,7 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	bootstrap.sh
-	BOOST_CPPFLAGS=-I/system/$relativeHearersDir runConfigure \
+	BOOST_CPPFLAGS=-I/system/$relativeHeadersDir runConfigure \
```

Comment:
That's a godd question. Well, I did not dare to remove it because I can't test the build against Qt5 (since I don(t have pre-build packages of Qt5 and building them requires a lot of free space and takes a very long time). But if you have tried without the BOOST_CPPFLAGS environment variable and found out that it still builds and runs fine then, of course, I can remove it in my commit ;-)